code/modules/projectiles/ammunition/bullets.dm

```diff
@@ -31,7 +31,7 @@
 /obj/item/ammo_casing/a10mmh
 	desc = "A 10mm high-velocity bullet casing."
 	caliber = "10mm"
-	projectile_type = /obj/item/projectile/bullet/hv/a10mm
+	projectile_type = /obj/item/projectile/bullet/a10mm/high_velocity
```

Comment:
no need tor fully writing out "/high_velocity", /hv will do fine

Reply:
Would this not go against the Naming conventions concerning abbreviations listed in the Contribution Guide? The object is not a variable within a proc.

Reply:
In this case, its a bit too unwieldy
Those conventions are largely for variable names, not typepaths.
Besides which, there are some real technical limitations with overlong paths

Reply:
Considering that, I'll abbreviate the path down to `/hv`.